robottelo/cli/factory.py

```diff
@@ -321,6 +325,115 @@ def make_content_view(options=None):
 
 
 @cacheable
+def make_content_view_filter(options=None):
+    """
+    Usage::
```

Comment:
i can't see any factory `make_***` method without usage docstring, and yes, it duplicates create docstring - few examples are `docker registry create` or `docker container create`. I don't mind, we can change this by dropping usage from `create()` method if you think there's a point in it.